crates/extensions/az_mapper_ext/src/converter.rs

```diff
@@ -2,18 +2,22 @@ use crate::error::ConversionError;
 use crate::size_threshold::SizeThreshold;
 use clock::Clock;
 use log::error;
+use serde_json::Map;
+use serde_json::Value;
 use std::convert::Infallible;
 use tedge_actors::Converter;
 use tedge_api::health::is_bridge_health;
-use tedge_api::serialize::ThinEdgeJsonSerializer;
+use tedge_api::mqtt_topics::Channel;
+use tedge_api::mqtt_topics::EntityTopicId;
+use tedge_api::mqtt_topics::MqttSchema;
 use tedge_mqtt_ext::MqttMessage;
 use tedge_mqtt_ext::Topic;
 
 const AZ_MQTT_THRESHOLD: usize = 1024 * 128;
 
 #[derive(Debug)]
 pub struct MapperConfig {
```

Comment:
I feel this `MapperConfig` is no longer useful. Changing `out_topic` to `String` is a sign. I would remove it and hard-code the topic in each function that requires these topics.

Reply:
removed